_config.yml

```diff
@@ -582,6 +582,14 @@ collections:
   platforms:
     output: false
 
+donation_banner:
+  address: 1GwV7fPX97hmavc6iNrUZUogmjpLPrPFoE
+  display: true
```

Comment:
This should also be used for conditionally loading other files (e.g. in index.html).